ui/perfherder/perf-helpers/perfdocs.js

```diff
@@ -93,63 +99,92 @@ const browsertimeBenchmarks = [
   'youtube-playback-widevine-hfr',
 ];
 
+const invertedTestsNamesDevtools = {
+  'parent-process:toolbox': 'toolbox:parent-process',
+  'parent-process:target': 'target:parent-process',
+  'parent-process:reload': 'reload:parent-process',
+  'content-process:reload': 'reload:content-process',
+};
+
+const removedOldTestsDevtools = [
+  'total-after-gc',
+  'reload-total-after-gc',
+  'content-total-after-gc',
+  'reload-content-total-after-gc',
+  'toolbox-total-after-gc',
+  'target-total-after-gc',
+];
+
 /**
  * TODO: remove hardcoded names once suffixes are removed from Perfdocs
  * @link https://firefox-source-docs.mozilla.org/testing/perfdocs/raptor.html#custom
  */
 const browsertimeCustomTests = ['process-switch', 'welcome'];
 
-const baseURL = 'https://firefox-source-docs.mozilla.org/testing/perfdocs/';
+const baseURL = 'https://firefox-source-docs.mozilla.org/';
 
 export class Perfdocs {
   /**
    * Class that provides a link where possible with
    * detailed information about each test suite.
    */
-  constructor(framework, suite, platform = null, title = null) {
-    this.framework = framework;
+  constructor(framework, suite, platform = '', title = '') {
+    this.framework = framework || '';
     this.suite = suite || '';
-    this.platform = platform;
+    this.platform = platform || '';
     this.title = title || '';
-    this.url = null;
-    this.remainingTestName = null;
+    this.remainingTestName = '';
   }
 
   get remainingName() {
-    if (!this.remainingTestName) {
+    if (this.remainingTestName.length === 0) {
       this.remainingTestName = this.title.replace(this.suite, '');
     }
     return this.remainingTestName;
   }
 
   get documentationURL() {
-    if (!this.url) {
-      const frameworkName = supportedPerfdocsFrameworks[this.framework];
-      this.url = baseURL.concat(
+    let url;
+    const frameworkName = supportedPerfdocsFrameworks[this.framework];
+    if (!frameworkName) {
+      url = baseURL.concat('testing/perfdocs/');
+      return url;
+    }
+    if (!url) {
+      url =
+        frameworkName !== 'performance-tests-overview'
```

Comment:
it's easier to read when the conditions are positive like `frameworkName === 'performance-tests-overview'`